apisupport.ant/src/org/netbeans/modules/apisupport/project/resources/layer.xml

```diff
@@ -261,11 +261,6 @@
                 <attr name="displayName" bundlevalue="org.netbeans.modules.apisupport.project.ui.Bundle#Templates/Licenses/license-cddl-netbeans-sun.txt"/>
                 <attr name="mavenLicenseURL" stringvalue="http://www.netbeans.org/cddl-gplv2.html"/>
             </file>
-            <file name="license-apache20-netbeans.txt" url="license-apache20-netbeans.txt">
```

Comment:
Should be fixed this.

Reply:
That line is deleted.

Reply:
Yes, should be fixed in the same module. i.e. Please change this line. (Please don't change `projectui`.)

Reply:
Not sure I understand. This part of `layer.xml` in `apisupport.ant` is no longer relevant and is therefore deleted. The license template file is now supplied from `projectui` module.

Reply:
Note : The license template file *must* be supplied by `projectui` module so that it can be used by any type of project which wishes to use it. In the old days the license header for CDDL NetBeans project lived in `apisupport.ant` because it was a very special template which didn't have relevance outside of NetBeans itself.  But this is different now. The ASF header is relevant for any project under the ASF, not just NetBeant itself. This is why `projectui` is now the provider of the ASF lic template, just like for any other generic lic template. I hope this makes sense.

Reply:
Ah, I see now, sorry.

Reply:
No worries. I think you need to remove something here so that GitHub doesn't continue to flag it as "changed requested".  Not sure how you do that.